packages/form/src/Form.js

```diff
@@ -41,7 +41,7 @@ Form.propTypes = {
   initialTouched: PropTypes.object,
   validationSchema: PropTypes.object,
   validate: PropTypes.func,
-  children: PropTypes.node,
+  children: PropTypes.oneOf([PropTypes.func, PropTypes.node]),
```

Comment:
Can you also update the `typescript` definition for this to work?

Example in `Spaces`:  https://github.com/Availity/availity-react/blob/master/packages/spaces/types/Spaces.d.ts#L82

Reply:
I think that is already done. `Form` [extends](https://github.com/Availity/availity-react/blob/db870082e5eb487c455963578390be6f6dfec816/packages/form/typings/Form.d.ts#L4) `React.Component` with `FormikConfig`, which [expects children as Function](https://github.com/jaredpalmer/formik/blob/f117c04738ed218b5eb8916d7189e0849962d50d/packages/formik/src/types.tsx#L178).

Reply:
Oh true 🤦‍♂ 

I forgot to notice we already extend the `FormikConfig`